package.json

```diff
@@ -219,18 +219,20 @@
     "babel-preset-react": "6.24.1",
     "copy-webpack-plugin": "4.3.1",
     "cross-env": "5.1.4",
-    "electron": "3.1.3",
-    "electron-builder": "20.38.2",
-    "electron-builder-squirrel-windows": "20.38.2",
+    "electron": "6.0.0",
+    "electron-builder": "21.2.0",
+    "electron-builder-squirrel-windows": "21.2.0",
     "electron-devtools-installer": "2.2.4",
-    "electron-rebuild": "1.8.2",
+    "electron-rebuild": "1.8.5",
     "eslint": "4.7.2",
     "eslint-config-prettier": "2.6.0",
     "eslint-plugin-prettier": "2.3.1",
     "eslint-plugin-react": "7.3.0",
     "husky": "0.14.3",
     "inquirer": "5.1.0",
-    "node-gyp": "3.6.2",
+    "node-gyp": "5.0.3",
+    "node-pty": "^0.8.1",
```

Comment:
@Stanzilla this is the problem, you need to get on one of the newer beta builds for Electron 6 support. Try `node-pty@0.9.0-beta26`

Reply:
@Tyriar hrm, still errors :(